celery/backends/redis.py

```diff
@@ -213,6 +213,7 @@ def __init__(self, host=None, port=None, db=None, password=None,
         socket_connect_timeout = _get('redis_socket_connect_timeout')
         retry_on_timeout = _get('redis_retry_on_timeout')
         socket_keepalive = _get('redis_socket_keepalive')
+        health_check_interval = _get('redis_backend_health_check_interval')
```

Comment:
since this is a new config we should document it as well

Reply:
Thank you for the review.  Newer to this project so I'm not 100% clear on documentation best practices but I took a stab at it by manually updating docs/userguide/configuration.rst in https://github.com/celery/celery/pull/6666/commits/6c7148b6ef00d27297780092371711380c097996.  Please let me know if there's a better way to accomplish this.